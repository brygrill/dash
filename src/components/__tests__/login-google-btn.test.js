import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import LoginGoogleComponent from '../login/login-google-btn';

describe('<LoginGoogleComponent />', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LoginGoogleComponent />, div);
  });
  it('Matches snapshot with child', () => {
    const component = renderer.create(<LoginGoogleComponent />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
