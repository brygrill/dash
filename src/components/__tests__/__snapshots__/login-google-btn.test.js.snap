exports[`<LoginGoogleComponent /> Matches snapshot with child 1`] = `
<div
  style={
    Object {
      "padding": "1rem 1rem 0 1rem",
    }
  }>
  <button
    className="ui large button"
    onClick={[Function]}
    style={
      Object {
        "backgroundStyle": "#d4d4d5",
      }
    }
    tabIndex={undefined}>
    <i
      aria-hidden="true"
      className="google icon" />
     
    Login with Google
  </button>
  <div
    className="ui horizontal divider"
    style={
      Object {
        "margin": 0,
        "paddingTop": "1rem",
      }
    }>
    Or
  </div>
</div>
`;
